src/components/user-stat-block/index.tsx

```diff
@@ -36,10 +36,17 @@ export default function UserStatBlock({ rank, userStats }: Props) {
 
 const $Content = styled.li`
   display: grid;
-  grid-gap: 2rem;
+  gap: var(--space-5);
   grid-template-columns: var(--space-6) 1fr 1fr;
   grid-template-areas: 'rank user stats';
   align-items: center;
+  @media screen and (max-width: 1024px) {
+    grid-template-columns: min-content 1fr;
+    grid-template-areas: 'rank user' 'stats stats';
```

Comment:
I feel like this would look a bit better if the rank was centered vertically and had the whole height to be centered in.

Something like so:
```suggestion
    grid-template-areas: 'rank user' 'rank stats';
```

Reply:
Further to this, it would then make sense to have the user block left-aligned when in mobile, too

Reply:
Okay. I was considering doing that, but couldn't decide which one looked better, so I just picked one :sweat_smile: